src/controllers/organizationController.ts

```diff
@@ -233,4 +246,41 @@ export default class OrganizationController {
       );
     });
   }
+
+  private async createSignedDocument(
+    inputPath: string,
+    outputPath: string
+  ): Promise<Option<Error>> {
+    try {
+      const unsignedContentBase64 = await fs.promises.readFile(inputPath, {
+        encoding: "base64"
+      });
+      const errorOrSignedContentBase64 = await this.documentService.signDocument(
+        unsignedContentBase64
+      );
+      return errorOrSignedContentBase64.fold(
+        async error => {
+          log.error("An error occurred while signing document. %s", error);
+          return some(error);
+        },
+        async signedContentBase64 => {
+          try {
+            await fs.promises.writeFile(outputPath, signedContentBase64, {
+              encoding: "base64"
+            });
+            return none;
+          } catch (error) {
+            log.error(
+              "An error occurred while saving signed document. %s",
+              error
+            );
+            return some(error);
+          }
+        }
+      );
+    } catch (error) {
+      log.error("An error occurred while reading unsigned document. %s", error);
+      return Promise.resolve(some(error));
```

Comment:
```suggestion
      return some(error);
```

since the function is async